### PR TITLE
Added support for IONODES video encoders

### PR DESCRIPTION
**Pull request by priiduonu**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/ionodes.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+mib: IF-MIB::IONODES-IONSERIES-MIB
```
Comment (PipoCanaja) [inline includes/definitions/discovery/ionodes.yaml]: You need only one ':' here and not two

**Comment by PipoCanaja** [comment]
Hi @priiduonu 
Thank you for your PR. A few comments inline !
Bye

File: includes/definitions/ionodes.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+os: ionodes
+text: 'IONODES'
+type: encoder
+icon: ionodes
```
Comment (PipoCanaja) [inline includes/definitions/ionodes.yaml]: mib directory for ionodes is missing here.

**Comment by priiduonu** [comment]
Thanks for Your comments @PipoCanaja ! Made the suggested changes.

File: includes/definitions/discovery/ionodes.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+mib: IF-MIB:IONODES-IONSERIES-MIB
```
Comment (PipoCanaja) [inline includes/definitions/discovery/ionodes.yaml]: You do not use IF-MIB, do you ? if not, you can remove it from here and only keep IONODES-IONSERIES-MIB

**Comment by priiduonu** [comment]
Keeps working :-)

**Comment by PipoCanaja** [comment]
If it is fine with your tests, we will be able to merge that next week after next release.

**Comment by PipoCanaja** [comment]
Release done, let's merge this.

**Pull request merged**